lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/externalcomponent/ExternalComponentViewController.java

```diff
@@ -87,6 +87,11 @@ public void applyBottomInset() {
         if (view != null) presenter.applyBottomInset(view, getBottomInset());
     }
 
+    @Override
+    public String getCurrentComponentName() {
```

Comment:
You mentioned that external component won't have component name. So should this return `null`?

Reply:
Return `externalComponent.name.get()` which is the name used to register the externalComponent with

Reply:
Now returning:

```java
@Override
public String getCurrentComponentName() {
  return externalComponent.name.get();
}
```